### ISSUE TITLE
Copy DashBoard

### ISSUE THREAD

**Issue by zraffaele** [datalust/seqcli#182]
Hi all,
if copy a dashboard from instance to another i have the errors that the signal-id is not find.
How can i copy entities from instance to another?
Thanks in advance

RZ

**Comment by nblumhardt** [comment]
Hi @zraffaele! Thanks for your message.

At the moment it's tricky - you would need to use the C# API client or similar code, and walk through all the properties of the `Dashboard`, replacing ids of signals on the source server with signals on the destination server.

We're aiming to address this with #176, which will enable exporting a set of related entities into a template, and then applying the template to the destination server. It would be great if you have a chance to add some details of your use case over there:

 * Would you expect the signals to already exist on the destination server?
 * Do you need to do this only once, or do you want to keep the destination server updated as the dashboard is changed on the source server?
 * Anything else that might shape how we design the feature?

Thanks!

**Comment by zraffaele** [comment]
Thanks for the answer.
In my opinion the best solution is that the exporter export all, dashboard and signals.
For the second question in my opinion it is not necessary to syncronize  the instances.
In general, what is missing is a module that exports a bit of everything, queries, signals, dashboards, configs, apps etc etc. 

Thanks !

RZ

**Comment by nblumhardt** [comment]
Thanks! I'll close this as a duplicate of #176, but if you need any help with the code-driven workaround in the meantime, please get in touch.
